runtime/recipe/recipe.js

```diff
@@ -28,6 +28,10 @@ class Recipe {
 
     // TODO: Change to array, if needed for search strings of merged recipes.
     this._search = null;
+
+    // The recipe as it was created or loaded from manifest. Each plan holds the initial version
+    // of recipe it was resolved from.
+    this._baseRecipe = null;
```

Comment:
Not used yet? I'm not sure whether we want should store this here, leaning towards having the the planner/strategies yield both the fully resolved recipe together with the generalized version.